pkgs/stdenv/linux/make-bootstrap-tools.nix

```diff
@@ -176,11 +189,14 @@ rec {
     bootstrapTools = runCommand "bootstrap-tools.tar.xz" {} "cp ${build}/on-server/bootstrap-tools.tar.xz $out";
   };
 
-  bootstrapTools = import ./bootstrap-tools { inherit system bootstrapFiles; };
+  bootstrapTools = import ./bootstrap-tools {
+    inherit (hostPlatform) system;
```

Comment:
This will now be built on the platform the stdenv's host platform. For native, this makes so difference, but for cross it does mean we would have permanently enqueued derivations, unless we add builders for those platforms.

Reply:
Added an assertion to prevent those jobs from appearing. Easy to undo if we want them later.